hadoop-hdds/framework/src/main/java/org/apache/hadoop/ozone/audit/AuditMarker.java

```diff
@@ -25,7 +25,8 @@
 public enum AuditMarker {
   WRITE(MarkerManager.getMarker("WRITE")),
   READ(MarkerManager.getMarker("READ")),
-  AUTH(MarkerManager.getMarker("AUTH")),;
+  AUTH(MarkerManager.getMarker("AUTH")),
+  PERFORMANCE(MarkerManager.getMarker("PERFORMANCE"));
```

Comment:
What is the intended use of this marker?

Reply:
Add a mark to better distinguish it from other audit logs.